apps/orders/views.py

```diff
@@ -54,44 +59,118 @@ def checkout(self, request):
                 price=cart_item.price,
             )
 
-        serializer = self.get_serializer(order)
-        ShoppingCart.objects.filter(user=request.user).delete()
-        return Response(serializer.data, status=status.HTTP_201_CREATED)
+        serializer = self.get_serializer(order)  # noqa
 
-    @action(detail=True, methods=['post'])
-    @transaction.atomic
-    def process_payment(self, request, pk=None):
-        order = self.get_object()
+        kobo_amount = int(total_cost * 1000)
+        # callback_url = request.build_absolute_uri(reverse("payment_verify"))
+        callback_url = request.build_absolute_uri(
+            "http://127.0.0.1:8000/api/order/payment/verify/"
+        )
+        # callback_url = reverse("payment_verify")
 
```

Comment:
why se this instead of `reverse`?

Reply:
I was getting some errors with reverse

Reply:
I'm still getting the "NoreverseMatch" error